### ISSUE TITLE
Readme issues

### ISSUE THREAD

**Issue by caioaao** [zmxv/react-native-sound#59]
There are some issues with the docs there:
## Missing imports

```
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;

import java.util.*;
```
## Wrong override

There is no `getPackages` on `ReactActivity` on RN 0.31, so the `@Override` should be removed.

**Comment by sdeleon28** [comment]
I think where it says `android/app/src/main/java/.../MainActivity.java`, you're actually supposed to edit `android/app/src/main/java/.../MainApplication.java` which extends `ReactApplication` instead. I'm creating a PR to fix that in the docs.

**Comment by sdeleon28** [comment]
Or maybe that's just in my particular config? I'm on RN 0.31.0

**Comment by caioaao** [comment]
Hm, idk. I got this code from [react-native-sound-demo](https://github.com/zmxv/react-native-sound-demo) and it worked (with RN 0.31.0 as well). I don't know if overriding `ReactActivity`'s `getPackages` is a good idea

**Comment by sdeleon28** [comment]
Not a clue. I'm pretty ignorant about native Android development. I follow instructions on the README like a monkey for the java part when I have to install a library like this :-)

In this case, `getPackages` wasn't in `MainActivity.java` for my project but was in `MainApplication.java` so I just edited it there. It seems to work OK although I'm unaware of the implications. Why do you think it's not a good idea? BTW, I created my project with `react-native init`.

If someone with a little more authority than me confirms editing `MainApplication.java` is actually the way to go, then I'll submit a PR for the README.

**Comment by benvium** [comment]
This has been fixed via PR #85
